prometheus/src/utils.rs

```diff
@@ -94,6 +94,11 @@ impl<'a> Metric<'a> {
             None => self,
         }
     }
+
+    pub fn with_suffix(mut self, suffix: &'a str) -> Metric<'a> {
+        self.suffix = suffix;
+        self
+    }
```

Comment:
Why is this change in here? It doesn’t appear to be used.

Reply:
It is used here (branch `prometheus-1.13`, `prometheus/src/utils.rs`)

https://github.com/ChorusOne/solana/blob/0dfa233d88fa38184339929d589db99f40a9568d/prometheus/src/utils.rs#L189-L199

Compilation errors otherwise

Reply:
Sorry, to clarify, errors when running `cargo check --tests`

Reply:
Ah, ok, :+1: then, ignore my comment!